tests/integration/helpers.py

```diff
@@ -316,12 +318,15 @@ async def execute_query_on_unit(
         password: The PostgreSQL superuser password.
         query: Query to execute.
         database: Optional database to connect to (defaults to postgres database).
+        sslmode: Optional ssl mode to use (defaults to None).
 
     Returns:
         A list of rows that were potentially returned from the query.
     """
+    extra_connection_parameters = f" sslmode={sslmode}" if sslmode is not None else ""
     with psycopg2.connect(
-        f"dbname='{database}' user='operator' host='{unit_address}' password='{password}' connect_timeout=10"
+        f"dbname='{database}' user='operator' host='{unit_address}'"
+        f"password='{password}' connect_timeout=10{extra_connection_parameters}"
```

Comment:
nit: I think is more legible to place the space of the string where you are adding things
```python
extra_connection_parameters = f"sslmode={sslmode}" if sslmode is not None else ""
f"password='{password}' connect_timeout=10 {extra_connection_parameters}"
```

Reply:
Yeah, I agree that we need more logging on our tests. I will improve on the next PRs. Thanks for the suggestions. I updated the code on https://github.com/canonical/postgresql-operator/pull/31/commits/83b95e8d9fb4506b62332761365afaecc087a20b.